doc-page-CSS/utilities.css

```diff
@@ -58,6 +58,10 @@ ul {
   border-radius: 2px;
 }
 
+.elem-round {
+  border-radius: 50%;
+}
+
```

Comment:
What is elem?
Write meaning classnames.

Reply:
Fixed this! Changed elem to element